src/renderer/components/TranscriptionBlock.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from '@emotion/styled';
+import { Box, colors } from '@mui/material';
+import { Transcription } from 'sharedTypes';
+
+const TranscriptionBox = styled(Box)`
+  background: ${colors.common.white};
+  height: 100%;
+  padding: 20px;
+`;
+
+const Word = styled('span')`
+  &:hover {
+    background: orange;
+  }
+`;
+
+interface Props {
+  transcription: Transcription;
+  onWordClick: (wordIndex: number) => void;
+}
+
+const TranscriptionBlock = ({ onWordClick, transcription }: Props) => {
+  return (
+    <TranscriptionBox>
+      <p>
+        {/* eslint-disable-next-line prettier/prettier */}
+        {transcription.words.map((word, index) => (
+          !word.deleted ? (
+            <Word
+              key={word.key.toString()}
+              data-index={index}
+              data-type="word"
+              onClick={() => {
+                onWordClick(index);
+              }}
+            >
+              {`${word.word} `}
+            </Word>
+          ) : null
+          /* eslint-disable-next-line prettier/prettier */
```

Comment:
What does this need to be disabled for?

Reply:
@fwang1703 @emmapainter

Reply:
Looks like the parentheses it was complaining about can actually be removed - for some reason before when we tried removing them it kept complaining, it might have just been super laggy to update though.